assets/stylesheets/common/chat-onebox.scss

```diff
@@ -20,8 +20,8 @@
       display: flex;
 
       .avatar {
-        aspect-ratio: 30 / 30;
         margin-right: 0.25rem;
+        max-width: 30px !important;
```

Comment:
<img width="517" alt="Screenshot 2022-06-10 at 16 13 55" src="https://user-images.githubusercontent.com/339945/173084880-2717374d-1b4c-4d5c-8cbb-4c169fcea75b.png">

Reply:
There's this one too:

<img width="255" alt="Screenshot 2022-06-10 at 16 15 36" src="https://user-images.githubusercontent.com/339945/173085236-6a884d71-712e-40a8-b33b-fcd03a40f658.png">

Sure removing it would fix it, but I have no idea why it's here in first place.

Reply:
After discussion we will try to remove width: auto from core first, https://github.com/discourse/discourse/pull/17059